test/tools/ossfuzz/protoToYul.h

```diff
@@ -32,6 +33,19 @@ namespace yul_fuzzer
 class ProtoConverter
 {
 public:
+	ProtoConverter()
+	{
+		m_numLiveVars = 10;
+		m_numVarsPerScope.push(m_numLiveVars);
+		m_numNestedForLoops = 0;
+	}
+	ProtoConverter(ProtoConverter const& _x)
```

Comment:
Do you actually need a copy constructor?

Reply:
No, I'll remove it